bin/main/log4j.properties

```diff
@@ -0,0 +1,8 @@
+# Root logger option
```

Comment:
The `bin` is a build directory and should be git ignored, similar to how we git ignore `build` (gradle uses `build/` by default, and Eclipse uses `bin` by default)

See the template [Eclipse.gitignore GitHub template](https://github.com/github/gitignore/blob/752b7678db497edb377fa982ded0c70035393794/Global/Eclipse.gitignore#L2):